src/Dapr.Configuration.Extensions/DaprSecretStoreConfigurationExtensions.cs

```diff
@@ -0,0 +1,58 @@
+// ------------------------------------------------------------
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+// ------------------------------------------------------------
+
+using System;
+using System.Net.Http;
+using Dapr.Client;
+using Microsoft.Extensions.Configuration;
+using Microsoft.Extensions.Configuration.DaprSecretStore;
+
+namespace Microsoft.Extensions.Configuration
+{
+    /// <summary>
+    /// Extension methods for registering <see cref="DaprSecretStoreConfigurationProvider"/> with <see cref="IConfigurationBuilder"/>.
+    /// </summary>
+    public static class DaprSecretStoreConfigurationExtensions
+    {
+        /// <summary>
+        /// Adds an <see cref="IConfigurationProvider"/> that reads configuration values from the Dapr Secret Store.
+        /// </summary>
+        /// <param name="configurationBuilder">The <see cref="IConfigurationBuilder"/> to add to.</param>
+        /// <param name="store">Dapr Secre Store name.</param>
+        /// <param name="secrets">The secrets to retrieve.</param>
+        /// <param name="client">Dapr client used to retrieve Secrets</param>
```

Comment:
Suggestion, remove some captalization from places like `Secrets` here, probably the same for `Dapr Secret Store` -> `Dapr secret store`